.cron.yml

```diff
@@ -10,10 +10,17 @@ jobs:
           target-tasks-method: l10n_screenshots
           treeherder-symbol: l10-screenshots
       when: []  # Manual trigger only
-    - name: tab-performance-test
```

Comment:
We would need to update this file too so that the cron job keeps working: 
https://hg.mozilla.org/ci/ci-configuration/file/26312c9dff1107eff338a231785db6d1947126fc/projects.yml#l581

I can help you to create the PR there if you are not familiar with that...

Reply:
And also we need to grant them permission: 
https://hg.mozilla.org/ci/ci-configuration/file/tip/grants.yml#l1779